content/docs/guides/crosswalk/aws/api-gateway.md

```diff
@@ -14,20 +14,20 @@ aliases: ["/docs/reference/crosswalk/aws/api-gateway/"]
     <img src="/images/docs/reference/crosswalk/aws/logo.svg" align="right" width="280" style="margin: 0 0 32px 16px;">
 </a>
 
-[Amazon API Gateway](https://aws.amazon.com/api-gateway/) is a fully managed service for creating, monitoring, and
+Pulumi Crosswalk for AWS provides significantly easier ways of programming an AWS API Gateway. This includes using
```

Comment:
I actually think this works well, but it's inconsistent with the rest of the pages in this section. So if we do apply this change here, we should probably do so on its siblings as well.